webpack.config.js

```diff
@@ -75,7 +75,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
             rules: [
                 {
                     test: /\.js$/,
-                    exclude: /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|lodash-es)[/\\])/,
+                    exclude: /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|lodash-es)[/\\])/,
```

Comment:
See https://github.com/ckeditor/ckeditor5/issues/9554 there seems to be a strange new core packages just called `ckeditor5` which also need to transpiled

Reply:
It was not required for the build but was afraid that something could go wrong when I only add it to jest and not also here